### ISSUE TITLE
axios 0.15 doesn't work with SystemJS

### ISSUE THREAD

**Issue by markwoon** [mzabriskie/axios#485]
This works with 0.14.0 but fails with 0.15.0:

```
 System.import("/js/vendor/axios.js")
        .then(function(module) {
          alert('hello');
        });
```

**Comment by mzabriskie** [comment]
Do you have an error message?

**Comment by markwoon** [comment]
Here's the error I get:
```
system.js:4 Uncaught (in promise) Error: (SystemJS) XHR error (404 Not Found) loading http://localhost/js/undefined.js
	Error: XHR error (404 Not Found) loading http://localhost/js/undefined.js
	Error loading http://localhost/js/undefined.js as "undefined" from http://localhost/js/vendor/axios.js
```

You can see it in action here: http://codepen.io/anon/pen/BLPPNJ?editors=1111

If you revert to 0.14.0 you'll get the alert.

**Comment by nickuraltsev** [comment]
Fixed in v0.15.1.
